dev-python/raven/raven-5.31.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_3,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Raven is a client for Sentry"
+HOMEPAGE="https://github.com/getsentry/raven-python"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=""
+DEPEND="
+	dev-python/pytest[${PYTHON_USEDEP}]
```

Comment:
you sure this is unconditionally needed? Usually this is only needed when running the testsuite, which is further made conditional with `IUSE="test"` and then put in a `test? ( )` block